### PR TITLE
testing cors

### PR DESCRIPTION
**Pull request by LeTurt333**

### CODE REVIEW

File: hooks/fetchUserStatus.ts [hunk 1]
```diff
@@ -36,6 +36,8 @@ export const fetchUserStatus = async ({
     });
     console.log(res);
     const resData = await res.json();
+    console.log("____")
+    console.log(resData);
```
Comment (webmaster128) [inline hooks/fetchUserStatus.ts:40]:
```suggestion
```

File: package-lock.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "randdrop-ui",
+  "name": "randdrop-frontend",
```
Comment (webmaster128) [inline package-lock.json:2]:
```suggestion
  "name": "randdrop-ui",
```

File: services/apiHelpers.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  },
+  "aura": {
+    //"url": 'https://gist.githubusercontent.com/kaisbaccour/05b6dda6672a6b9c7beaf6198006a4c2/raw/11cbf1f49dba15bfd07f61ea4430392f556441d0/gistfile1.json',
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.2.1/aura-randdrop-2.json',
+    "contract": 'aura1femc3zxv4c43rwpy43na9gul4ssr8dkxe3m9v8afzlmhgj69sw6ql28x39'
+  },
+  "osmosis": {
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.2.1/osmosis-randdrop-1.json',
+    "contract": 'osmo1873ls0d60tg7hk00976teq9ywhzv45u3hk2urw8t3eau9eusa4eqtun9xn'
+  }
```
Comment (webmaster128) [inline services/apiHelpers.ts:58]: Should we remove those URLs? They are not used anymore now

Comment (LeTurt333) [reply services/apiHelpers.ts:58]: Good call

File: services/apiHelpers.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  winning_amount: z.string().optional(),
+  claim_contract: z.string().optional()
+}).strict()
+export type CheckResponse = z.infer<typeof CheckResponse>;
+
+
+
+export const AirdropRegistry = {
+  "juno": {
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.2.1/juno-randdrop-2.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:35]:
```suggestion
```

File: services/apiHelpers.ts [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    "contract": 'juno1pqcxmuc4nhkjqgymwraxzqj25yljdlnmu7rd675c6ypg9dakcwlszvk9gk'
+  },
+  "injective": {
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.3.0/injective-randdrop-2.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:39]:
```suggestion
```

File: services/apiHelpers.ts [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    //"url": 'https://gist.githubusercontent.com/kaisbaccour/c26ede9d4219896bc03fb0fdfce310a3/raw/fa0825eb75607afad08ffb0fdead8567795150ad/injective-randdrop.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:40]:
```suggestion
```

File: services/apiHelpers.ts [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+     "contract": 'inj1z4nmwyaujc89tqwfhd6wm6hugdr3f4jnllxfgk'
+    // for testing
```
Comment (LeTurt333) [inline services/apiHelpers.ts:42]:
```suggestion
```

File: services/apiHelpers.ts [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    //"url": 'https://gist.githubusercontent.com/hoangdv2429/f4569307b2191e5a40d468019297a35f/raw/bb5560436bb2f747393503fdc9a25c8327166d4e/injective-randdrop.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:43]:
```suggestion
```

File: services/apiHelpers.ts [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+    //"contract": 'inj1hjl70antdrqlhunaesp8d6lv4nuy9rknfpn48w'
+  },
+  "stargaze": {
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.2.1/stargaze-randdrop-2.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:47]:
```suggestion
```

File: services/apiHelpers.ts [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+    "contract": 'stars1qytke02t6mrj6zvyjymrd4e6h3l78ecx2nwwf9huvcprehy7xmsqpqh7za'
+  },
+  "aura": {
+    //"url": 'https://gist.githubusercontent.com/kaisbaccour/05b6dda6672a6b9c7beaf6198006a4c2/raw/11cbf1f49dba15bfd07f61ea4430392f556441d0/gistfile1.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:51]:
```suggestion
```

File: services/apiHelpers.ts [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.2.1/aura-randdrop-2.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:52]:
```suggestion
```

File: services/apiHelpers.ts [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+    "contract": 'aura1femc3zxv4c43rwpy43na9gul4ssr8dkxe3m9v8afzlmhgj69sw6ql28x39'
+  },
+  "osmosis": {
+    "url": 'https://raw.githubusercontent.com/noislabs/randdrop-snapshots/v0.2.1/osmosis-randdrop-1.json',
```
Comment (LeTurt333) [inline services/apiHelpers.ts:56]:
```suggestion
```

**Pull request merged**